### PR TITLE
styling landing page

### PR DESCRIPTION
**Pull request by brentmartin**
@jessmartin

### CODE REVIEW

**Comment by jessmartin** [comment]
:+1: looks good to me

File: db/seeds.rb [hunk 1]
```diff
@@ -6,26 +6,7 @@
...
-#   end
-# end
-#
-# # Surveys
-# 8.times do
-#   Survey.create(heading: Faker::StarWars.quote, intro: Faker::Hipster.paragraph(rand(2..5)), assessment_id: rand(1..Assessment.count))
-# end
 
-# Create a user (not yet!)
-# Admin will create an assessment (template) that will be viewed by potential assessees
```
Comment (jessmartin) [inline db/seeds.rb:28]: Yay deleting commented out code!

**Comment by brentmartin** [comment]
Hows it looking now?

@jessmartin

**Comment by jessmartin** [comment]
Haven't had a chance to test it locally, but scanning through the code, looks good.

:ship: it!

**Pull request merged**